test_apis

pranavv1 opened the pull request.

pranavv0 committed:
added test

pranavv1 committed:
Create test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv1 commented:
/test_rs

pranavv0 commented:
@pranavv1 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv1 commented:
/test_rs

pranavv0 commented:
@pranavv1 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv1 commented:
/test_rs

pranavv0 commented:
@pranavv1 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
/test_rs

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
/test_rs

pranavv0 commented:
@pranavv0 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

pranavv0 commented:
/test_rs

karun-singh committed:
added Rancher RKE2 setup guide

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 committed:
Create test_common.yml

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_gis

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_cat

Commit:
K8s-cat : Added mlayer and postgres tables

Commit:
K8s-cat : Added resource values for mlayer and postgres

Commit:
K8s-cat : updated values.yaml  for mlayer and postgres

Commit:
K8s-cat : Added templates/mlayer

Commit:
K8s-cat : Added templates/postgres

Commit:
K8s-cat : Reformated apiServer table in README.md

abhi4578 committed:
Update K8s cluster directory

- rearrange directories to make more sense
- add Readme at each sub-directory level

abhi4578 committed:
Update README.md

pranavv0 committed:
updated test code

abhi4578 committed:
Correct resource values of cat, inc. chart version

abhi4578 committed:
Merge pull request #384 from pranavv0/lip-4.5.3

updated lip test code

jenkins-datakaveri committed:
updated cat docker image tag to 4.5.0-alpha-d1a107e (#382)

Co-authored-by: Abhi <29863835+abhi4578@users.noreply.github.com>

jenkins-datakaveri committed:
updated AAA docker image tag to 4.5.0-alpha-124b88b (#385)

jenkins-datakaveri committed:
updated file-server docker image tag to 4.5.0-alpha-d52c02b (#368)

jenkins-datakaveri committed:
updated gis docker image tag to 4.5.0-alpha-b0b5a03 (#372)

jenkins-datakaveri committed:
updated lip docker image tag to 4.5.alpha-d5f7eb8 (#386)

jenkins-datakaveri committed:
updated rs docker image tag to 4.5.0-alpha-3b6dd1e (#363)

abhi4578 committed:
Upgrade API servers to 4.5.0 (#387)

* upgrade di, correct cat nlp config

* Upgrade rs helm and stack, di stack tag

jenkins-datakaveri committed:
updated cat docker image tag to 4.5.0-alpha-59b38b9 (#388)

abhi4578 committed:
update rs, fixes spatial, adapter table permission

dependabot[bot] committed:
Bump redis in /Docker-Swarm-deployment/single-node/lip/tests (#391)

Bumps [redis](https://github.com/redis/redis-py) from 4.5.3 to 4.5.4.
- [Release notes](https://github.com/redis/redis-py/releases)
- [Changelog](https://github.com/redis/redis-py/blob/master/CHANGES)
- [Commits](https://github.com/redis/redis-py/compare/v4.5.3...v4.5.4)

---
updated-dependencies:
- dependency-name: redis
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

abhi4578 committed:
Resource server fixes

- have 1 cache verticle instance, hpa disabled
- update resource values to not have fractional bytes

pranavv1 commented:
/test_cat

pranavv0 commented:
@pranavv1 You are not authorized to trigger this workflow. Only users with write permission can trigger this workflow.

abhi4578 committed:
RMQ fixes

- whole number resource values
- upgrade rabbitmq
- add extra permissions to to dx-di-user

abhi4578 committed:
immudb fixes and simplify  k8s gitignore

- annotate pods for prometheus scraping, update the correct port
- one .gitignore at top directory is enough

karun-singh committed:
migrate ca image registry from k8s.gcr.io to registry.k8s.io (#406)

abhi4578 committed:
reduced memoery,cpu for redis

abhi4578 committed:
disable cache pod hpa

abhi4578 committed:
update psql resource values

- use whole number requests and limits
- increase disk size as it will also be
  used for auditing

abhi4578 committed:
uodate fs, cat resource values

- make resource values whole numbers
- correct node-selector file server aws resources
- disable hpa for profanitychecksdk cat pod

abhi4578 committed:
update resource values of mon-stack

- add retention limit in loki to 30 days
- whole number resource values
- disable self monitoring, deployment of operator in loki

abhi4578 committed:
add rmq init-rmq job in install

- init rmq job is added to install as
 it waits till rmq is ready and then creates
 the necessary configuration

abhi4578 committed:
Merge pull request #400 from abhi4578/4.5.0-post-release-fixes

Master: 4.5.0 post release fixes

abhi4578 committed:
upgrade api servers to 5.0.0-alpha image tags (#415)

jenkins-datakaveri committed:
updated rs docker image tag to 5.0.0-alpha-67697d9 (#423)

abhi4578 committed:
Update swarm-setup.md

- correct default subnet address
- add notes

abhi4578 committed:
Fix unable to deploy encryption verticle in docker

- Error log while encyrption verticle deployment -
  ``java.lang.NoClassDefFoundError: Could not initialize class
com.sun.jna.Native``
- Happened because of use of tmpfs
   - libsodium libraries fail to load in Docker when --tmpfs /tmp
     due to "noexec"  flag on /tmp see:
     1. https://github.com/netty/netty-tcnative/issues/264
     2. https://github.com/moby/moby/issues/18853

abhi4578 committed:
in rs k8s encryption, use ssd based emptydir

- Use default/disk (denoted by empty string in medium) as emptyDir
  medium because
  - untar of fatjar is done by encryption vertcile to
  load classes and considerable size
  - hence use default medium instead of Memory, as it would consume
   lot of Memory for the pod

abhi4578 committed:
Merge pull request #424 from abhi4578/4.5.0-post-release-fixes

4.5.0 post release fixes II

abhi4578 committed:
increase postgres max-conn, fix logstash pipeline

- postgres max connections increased as more no. of components use it
- Logstash fix pipeline, by removing event field in docker
  see issue #193

abhi4578 committed:
correct rs-proxy, gitginore files

- use unamed volume for rs-proxy instead of tmpfs due to encryption
  verticle, see PR #424
- add gitignore file in rs-proxy, delete gitignored file

abhi4578 committed:
Add example-configs and remove gitignored nginx/conf

- need to have clear-site-data header directive with quotes
  and hence need to escape quotes ,
  ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data#directives

abhi4578 committed:
Correct Clear-Site-Data headers in K8s ingress

- need to have clear-site-data header directive with quotes
  and hence need to escape quotes ,
  ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Clear-Site-Data#directives

abhi4578 committed:
Remove Cache-Site-Data header, issue #426

abhi4578 committed:
simplify gitignore in Docker swarm

- merge all gitignore to one at top level

abhi4578 committed:
Simplify K8s gitignore

- merge gitignores to one in k8s deployment

abhi4578 committed:
merge gitgnore of swarm,K8s to one

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

abhi4578 committed:
Have one instance of CacheVerticle

- Many instances or pods of CacheVerticle can cause
  some sort of conflicts

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

abhi4578 committed:
rs-proxy config update

- add basepaths
- add audience field, adaptorReplyQueue

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_rs

THARAK-RAM1 committed:
K8s: added security components to rs-proxy (#445)

Co-authored-by: tharak1242 <tharak.ram@datakaveri.org>

abhi4578 committed:
Update RMQ with updated secrets structure

abhi4578 committed:
Update packages, docker install

- Update from Compose V1 to Compose V2, as V1 is depreceated and
  eol by June. See:  https://www.docker.com/blog/new-docker-compose-v2-and-v1-deprecation/
- Installation script changed according to :
  https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository

abhi4578 committed:
Merge branch 'master' of github.com:datakaveri/iudx-deployment into 4.5.0-post-release-fixes

abhi4578 committed:
Move adv-monstack dir as its not K8s addons

- Adv monstack is IUDX specific

abhi4578 committed:
Merge pull request #446 from abhi4578/4.5.0-post-release-fixes

4.5.0 post release fixes III

abhi4578 committed:
minor fix in docker install

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

pranavv0 commented:
/test_cat

abhi4578 committed:
Updated the index/repo readme

- added general overview deployment architecture figure
- update readme
   - overview of platform and deployment
   - features of deployment

abhi4578 committed:
Update K8s deployment README

- Add archtecture diagram, features, deployment ordering
- overview of K8s based deployment

karun-singh committed:
fix es-autoscaler rbac and script

(cherry picked from commit 984346b7b13ff9d1d3d26cd049dd37f92b0a0a15)

abhi4578 committed:
Correct grammar in main readme

- correct the reference to resource access services

abhi4578 committed:
Update K8s deployment Readme

- correct grammar

abhi4578 committed:
Add Swarm readme

- added overview deployment, features

abhi4578 committed:
Merge pull request #457 from karun-singh/fix-es-autoscaler

ES autoscaler fix

jenkins-datakaveri committed:
updated rs docker image tag to 5.0.0-alpha-59e4a24 (#459)

abhi4578 committed:
Merge pull request #453 from abhi4578/docs/readme

Update Readme

jenkins-datakaveri committed:
updated rs docker image tag to 5.0.0-alpha-b7af427 (#472)

jenkins-datakaveri committed:
updated FS docker image tag to 5.0.0-alpha-b7ecce0 (#470)

jenkins-datakaveri committed:
updated di docker image tag to 5.0.0-alpha-82821b5 (#473)

jenkins-datakaveri committed:
updated CAT docker image tag to 5.0.0-alpha-44b1c2a (#467)

pranavv0 committed:
Test Apis Workflow

pranavv0 committed:
Workfile

pranavv0 commented:
/test_rs

pranavv0 commented:
/test_cat

pranavv0 committed:
Added Readme and comments

pranavv0 commented:
/test_lip

pranavv0 commented:
/test_lip

pranavv0 commented:
/test_lip

pranavv0 committed:
added Lip

pranavv0 committed:
test script using pytest

pranavv0 committed:
update with requirement, Readme and script

pranavv0 committed:
Readme

pranavv0 committed:
one test script

pranavv0 committed:
deleted unused script

pranavv0 committed:
changes in workflow for lip

pranavv0 committed:
Changes requested with amqps

pranavv0 commented:
/test_lip

pranavv0 commented:
/test_lip

pranavv0 committed:
Merge branch 'apis-test' of https://github.com/pranavv0/iudx-deployment into test_rs

pranavv0 committed:
Merge branch 'test_rs' of https://github.com/pranavv1/iudx-deployment into test_rs

pranavv1 committed:
Merge branch 'pranavv0:master' into test_rs

pranavv0 committed:
changed req

pranavv0 commented:
/test_lip
